sharkFinApi/sharkFinApi/Controllers/UsersController.cs

```diff
@@ -20,6 +23,10 @@ public class UsersController : ControllerBase {
             _userRepository = userRepository;
             _portfolioRepository = portfolioRepository;
         }
+        public UsersController(IUserRepository userRepository)
```

Comment:
You don't need to overload constructors again. For testing you can just Moq the IPortfolioRepository or give it null if you aren't testing something that needs it. The real Controller requires a portfolio repository so you shouldn't add a constructor that gives the option of just ignoring it.